Doc/library/sqlite3.rst

```diff
@@ -2358,6 +2330,84 @@ can be found in the `SQLite URI documentation`_.
 .. _SQLite URI documentation: https://www.sqlite.org/uri.html
 
 
+.. _sqlite3-row-factory-how-to:
+
+How to work with row factories
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+By default, :mod:`!sqlite3` represent fetched rows as :class:`tuples <tuple>`.
+If a :class:`!tuple` does not suit your needs,
+use the built-in :class:`Row` type or a custom :attr:`~Connection.row_factory`.
```

Comment:
I remember discussing that wording with CAM and Ezio in a previous sqlite3 docs PR, and for some reason we stuck to "built-in". I'll see if I can dig up that conversation.

Reply:
I can see that "built-in" _may_ cause confusion. I'm not sure there's a good drop-in replacement word, but I'll try to reformulate that whole sentence.

Reply:
```suggestion
use the :class:`Row` class provided by the module or a custom :attr:`~Connection.row_factory`.
```

Reply:
Yeah, I can't remember that conversation off the top of my head, but I agree we shouldn't mix terminology like this.

However, instead of a rather wordy clarification, you could instead just include the module name in the reference title, which is more direct and concise. I'd also be consistent and specific, and call it a "class" rather than a generic type. So:

```suggestion
use the :class:`sqlite3.Row` class or a custom :attr:`~Connection.row_factory`.
```